content/source/docs/cloud/registry/using.html.md

```diff
@@ -71,9 +63,12 @@ Use the "Examples" dropdown to navigate to example modules and use the  Readme/I
 
 ![Terraform Cloud screenshot: a module submodules detail page](./images/using-module-examples.png)
 
-## Using Private Modules in Terraform Configurations
+## Adding Private Modules to Configurations
```

Comment:
@laurapacilio, I believe that there are a couple of ways of adding modules:
- VCS
- API

Would that be possible to consider breaking down this section into two methods, VCS and API?
Also, there is an option to use TFE provider as well where customers/users could use the [tfe_registry_module](https://registry.terraform.io/providers/hashicorp/tfe/latest/docs/resources/registry_module) to configure and manage modules in MR. Some customers/users also use [APIs](https://www.terraform.io/docs/cloud/api/modules.html#publish-a-private-module-from-a-vcs) to manage modules without VCS.

Reply:
We do have another page that talks about how to manage modules in the registry. It goes through the VCS workflow in the UI and has a note to point users to the API information you mentioned: https://www.terraform.io/docs/cloud/registry/publish.html.

This particular section is about defining/using modules in your configuration files - we give them example syntax and talk about how to format the various arguments. But I changed the title and now I feel like maybe it's unclear that it's about using modules in configurations, rather than managing them in the registry. Do you think it would make it clearer to change this title back to "Using" rather than "Adding"? Or we could say "Defining Private Modules in Configurations". 

What do you think about that?